pkgs/games/cataclysm-dda/wrapper.nix

```
@@ -28,5 +28,21 @@ else symlinkJoin {
         wrapProgram $out/bin/cataclysm-tiles \
             --add-flags "--datadir $out/share/cataclysm-dda/"
     fi
+
+    # Launch the wrapped proglam
```

```suggestion
    # Launch the wrapped program
```

you need to correct the commit message also

---

Sorry. Fixed.